core/iwasm/fast-jit/fe/jit_emit_memory.c

```diff
@@ -468,11 +468,11 @@ jit_compile_op_f64_store(JitCompContext *cc, uint32 align, uint32 offset)
 }
 
 bool
-jit_compile_op_memory_size(JitCompContext *cc, uint32 mem_idx)
+jit_compile_op_memory_size(JitCompContext *cc)
```

Comment:
How about not changing here, we might need to implement multiple memory support in the future?
How about just passing 0 in the frontend? jit_compile_op_memory_size(cc, 0)